certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -219,6 +228,33 @@ func (p *DeployableByOlmCheck) setUp(ctx context.Context, operatorData OperatorD
 	return nil
 }
 
+func (p *DeployableByOlmCheck) generateOperatorGroupData(operatorData *OperatorData) (cli.OperatorGroupData, error) {
+	var installMode string
+	for i := 0; i < len(prioritizedInstallModes); i++ {
+		if _, ok := operatorData.InstallModes[prioritizedInstallModes[i]]; ok {
+			installMode = prioritizedInstallModes[i]
+			break
+		}
+	}
+	log.Debug(fmt.Sprintf("The operator install mode is %s", installMode))
+	targetNamespaces := make([]string, 2)
+
+	switch installMode {
+	case string(operatorv1alpha1.InstallModeTypeOwnNamespace):
+		targetNamespaces = []string{operatorData.InstallNamespace}
+	case string(operatorv1alpha1.InstallModeTypeSingleNamespace):
+		targetNamespaces = []string{operatorData.TargetNamespace}
+	case string(operatorv1alpha1.InstallModeTypeMultiNamespace):
+		targetNamespaces = []string{operatorData.TargetNamespace, operatorData.InstallNamespace}
+	case string(operatorv1alpha1.InstallModeTypeAllNamespaces):
+		targetNamespaces = []string{}
+
+	}
+	log.Debug(fmt.Sprintf("The operator group's targetNamespaces is %s", targetNamespaces))
```

Comment:
Should this be `%v` since it is a slice? Or some form to get all the values from the slice?

Reply:
%s works well with string slice. Found this in the fmt doc:
```%s	the uninterpreted bytes of the string or slice
```

Reply:
That doesn't seem to be the case based on the log you shared. I might be missing something though. I would expect the below to have the values of the slice.
```
 time="2021-11-05T15:12:42Z" level=debug msg="The operator group's targetNamespaces is []"
 ```

Reply:
In the case of installMode: AllNamespaces, the target namespace of the operator group is empty. That's why you see `[]` in the output. Here is a sample output if the install mode is OwnNamespace:
```time="2021-11-05T16:54:28Z" level=debug msg="The operator install mode is OwnNamespace"
time="2021-11-05T16:54:28Z" level=debug msg="The operator group's targetNamespaces is [gitlab-runner-operator]"```